options/cf_options.cc

```diff
@@ -74,7 +74,12 @@ ImmutableCFOptions::ImmutableCFOptions(const ImmutableDBOptions& db_options,
       row_cache(db_options.row_cache),
       max_subcompactions(db_options.max_subcompactions),
       memtable_insert_with_hint_prefix_extractor(
-          cf_options.memtable_insert_with_hint_prefix_extractor.get()) {}
+          cf_options.memtable_insert_with_hint_prefix_extractor.get()),
+      cf_paths(cf_options.cf_paths) {
+  if(cf_paths.empty()) {
+    cf_paths = db_options.db_paths;
```

Comment:
Should this logic be in `SanitizeOptions` in "column_family.cc"? I think that'd be more consistent and simply other logic in `SanitizeOptions` (the `multiple_db_paths` computation).